core/src/main/java/io/grpc/internal/AbstractStream.java

```diff
@@ -305,7 +304,6 @@ public final void setDecompressor(Decompressor d) {
    * @param messageEncoding the name of the encoding provided by the remote host
    * @throws IllegalArgumentException if the provided message encoding cannot be found.
    */
-  @Override
   public final void setDecompressor(String messageEncoding) {
```

Comment:
It cannot, since it is used in the netty stream.  It cannot be accessed from netty handler.